Libs/Widgets/ctkErrorLogWidget.h

```diff
@@ -49,7 +50,13 @@ class CTK_WIDGETS_EXPORT ctkErrorLogWidget : public QWidget
   /// \sa ctkErrorLogModel::ColumnsIds
   Q_INVOKABLE void setColumnHidden(int columnId, bool hidden) const;
 
+  /// This property describes how the list of errors and error description are organized.
+  /// The orientation must be Qt::Horizontal (the default) or Qt::Vertical.
```

Comment:
For me, horizontal means the list and detail widget are in a horizontal layout (side-by-side). But if the default is horizontal then it means that now the default behaior is different, which we probably don't want. Please clarify in the method description what you mean by horizontal orientation (the widgets are side-by-side or above-under) and make sure that the default layout is the old layout (vertical, above-under).